src/darwin/Framework/CHIP/ServerEndpoint/MTRServerCluster.mm

```diff
@@ -165,11 +221,23 @@ - (BOOL)addAttribute:(MTRServerAttribute *)attribute
     }
 
     [_attributes addObject:attribute];
-    attribute.parentCluster = app::ConcreteClusterPath(_parentEndpoint, static_cast<ClusterId>(_clusterID.unsignedLongLongValue));
+    attribute.parentCluster = ConcreteClusterPath(_parentEndpoint, static_cast<ClusterId>(_clusterID.unsignedLongLongValue));
     return YES;
 }
 
-- (BOOL)associateWithController:(MTRDeviceController *)controller
+#define MTR_DECLARE_LIST_ATTRIBUTE(attrID) \
```

Comment:
Nit: the helper macro seems kinda low value compared to just repeating `ARRAY, 0, 0` 4 times and not having to define and undefine an extra macro

Reply:
Fair.  Will remove.

Reply:
Done.